README.md

```diff
@@ -64,23 +64,13 @@ pip install kymatio
 Linux and macOS are the two operating systems that are officially supported by Kymatio.
 
 
-### Available backends: PyTorch and scikit-cuda
-
-Kymatio is designed to interoperate on a variety of backends for array operations.
-The user may control the choice of backend at runtime by setting the environment variable `KYMATIO_BACKEND`, or by editing the Kymatio configuration file (`~/.config/kymatio/kymatio.cfg` on Linux).
-
-At the time of alpha release, the two available backends are PyTorch (`torch`) and scikit-cuda (`skcuda`) for 1D scattering and 2D scattering, and PyTorch only for 3D scattering.
-
-PyTorch is the default backend in 1D, 2D, and 3D scattering. Yet, for applications of the 2D scattering transform to large images (e.g. ImageNet, of size 224x224), we recommend the scikit-cuda backend, which is substantially faster than PyTorch.
-
-
 ### GPU acceleration
 
 
 To run Kymatio on a graphics processing unit (GPU), you should install the CUDA library and install the scikit-cuda dependency by running the following pip command:
 
 ```
-pip install -r requirements_optional_cuda.txt
+pip install -r https://raw.githubusercontent.com/kymatio/kymatio/master/requirements_optional.txt
```

Comment:
why not 
`pip install -r https://raw.githubusercontent.com/kymatio/kymatio/master/requirements_optional_cuda.txt` ?
(note the `_cuda` suffix)

Reply:
wow gosh, my bad

Reply:
btw why are you linking to the file online and not the local one? Not sure that is common practice

Reply:
What is the path without cloning it?!

Reply:
another, really way to do it (for the time being) is
```
pip install scikit-cuda cupy
```

Reply:
we have so few GPU-related dependencies for the moment that it might be overkill to write a `requirements.txt` file just for two lines

Reply:
Solved